docs_manual/developer_installation.rst

```diff
@@ -23,7 +23,7 @@ Install Postgres
 Follow the instructions for your operating system to install `Postgres <https://www.postgresql.org>`_.
 For Ubuntu, this would be::
 
-    sudo apt install postgresql
+    sudo apt install postgresql-12
```

Comment:
Please rather add a note about this. Ubuntu 20 includes postgresql v12 as `postgresql` package.